poetry/utils/exporter.py

```diff
@@ -14,19 +16,18 @@ def __init__(self, lock):  # type: (Locker) -> None
         self._lock = lock
 
     def export(
-        self, fmt, cwd, with_hashes=True, dev=False
-    ):  # type: (str, Path, bool, bool) -> None
+        self, fmt, cwd, output=None, with_hashes=True, dev=False
+    ):  # type: (str, Path, str, bool, bool) -> None
```

Comment:
It's actually an `Optional[str]`

Reply:
Learned something new again; thank you! Fixed with commit https://github.com/sdispater/poetry/pull/1035/commits/36cb4300894f9fb6114971df1b546db31fb46f4e.